importing/importbuttons.py

```
@@ -38,6 +38,7 @@ def import_pmx_models(directory):
                     files=[{'name': pmx_path}],
                     directory=pmx_path,
                     scale=1,
+                    clean_model = False,
```

@FlailingFog 

This is so that, if cards are exported incorrectly, instead of silently failing and letting mmd_tools try fix the model, mmd_tools will throw an error instead.